src/Traversal/Sdk/Sdk.targets

```diff
@@ -111,47 +126,67 @@
     <MSBuild Projects="@(ProjectReference)"
              BuildInParallel="$(BuildInParallel)"
              SkipNonexistentProjects="$(SkipNonexistentProjects)"
-             SkipNonexistentTargets="$(SkipNonexistentTargets)" />
+             SkipNonexistentTargets="$(SkipNonexistentTargets)"
+             ContinueOnError="$(ContinueOnError)" />
+    
+    <!-- If we ErrorAndContinue we need to propagate the error. -->
+    <Error Condition="'$(ContinueOnError)' == 'ErrorAndContinue' and '$(MSBuildLastTaskResult)' == 'false'" />
   </Target>
 
   <Target Name="Clean"
           DependsOnTargets="$(CleanDependsOn)"
           Condition=" '$(IsGraphBuild)' != 'true' ">
     <MSBuild Projects="@(ProjectReference)"
              Targets="Clean"
-             BuildInParallel="$(BuildInParallel)"
+             BuildInParallel="$(CleanInParallel)"
```

Comment:
I have this aversion for creating properties which can be defaulted in line since its just more strings.  Can you change this to:
```xml
  BuildInParallel="$([MSBuild]::ValueOrDefault('$(CleanInParallel)', '$(BuildInParallel)')"
```
That's effectively the same thing and lets you get rid of all the new properties being added if you use this pattern.  Logs still show what value is used so its not any harder to debug.

Reply:
done :) thanks for sharing the pattern. that's super helpful.